packages/aragon-wrapper/.babelrc

```diff
@@ -13,13 +13,14 @@
         },
         "useBuiltIns": false
       }
-    ],
-    "stage-2"
+    ]
   ],
   "plugins": [
-    ["inline-json-import", { "extensions": [".json"] }],
-    "transform-flow-strip-types",
-    "transform-runtime"
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-object-rest-spread",
```

Comment:
I've opted just to include the proposed features we use, rather than use the deprecated stage presets.